fvtr/fvtr.sh

```diff
@@ -197,20 +306,27 @@ do
 			case "$rc" in
 				"0")
 					echo -e "\tpassed"
-					total_passed=$(expr ${total_passed} + 1) ;;
+					total_passed=$(expr ${total_passed} + 1)
+
+					if [[ $LOUD = "OFF" ]];
+					then
+						echo -e "$i\tpassed" >> report.log
+					fi ;;
 				"38")
 					echo -e "\tignored"
-					total_ignored=$(expr ${total_ignored} + 1) ;;
+					total_ignored=$(expr ${total_ignored} + 1)
+
+					if [[ $LOUD = "OFF" ]];
+					then
+						echo -e "$i\tignored" >> report.log
```

Comment:
You don't need to duplicate code, you can always print the $i with "echo -n" and only print the status of failed tests.